play-validations/memory-footprint/src/main/java/com/google/wear/watchface/dfx/memory/AndroidResource.java

```diff
@@ -91,7 +91,8 @@ static AndroidResource fromPath(String filePath, byte[] data) {
     }
 
     static Boolean isValidResourcePath(Path filePath) {
-        Matcher m = VALID_RESOURCE_PATH.matcher(filePath.toString());
+        String pathWithFwdSlashes = filePath.toString().replace('\\', '/');
```

Comment:
Not that it matters, but I wonder if you should only do this if on windows, or when using the filesystem and separator = "\\".

But not blocking.